src/app/components/food-counter/food-counter.component.ts

```diff
@@ -0,0 +1,32 @@
+import { Component, OnInit } from '@angular/core';
+import { TreeNode } from '@circlon/angular-tree-component';
+import { TreeService } from 'src/app/services/tree.service';
+
+type PartialTreeNode = Partial<TreeNode> & {
+  name?: string;
+};
+
+@Component({
+  selector: 'app-food-counter',
+  templateUrl: './food-counter.component.html',
+  styleUrls: ['./food-counter.component.scss'],
+})
+export class FoodCounterComponent implements OnInit {
+  count = 0;
+
+  constructor(private treeSvc: TreeService) {
+    this._recursivelyCountNodes(
+      (this.treeSvc.getNodes() as unknown) as PartialTreeNode[]
+    );
+  }
+
+  ngOnInit(): void {}
+
+  /* Count number of nodes that don't start with "Tag: " */
+  private _recursivelyCountNodes = (nodes: PartialTreeNode[]) => {
+    nodes?.forEach((node: PartialTreeNode) => {
+      !node.name.startsWith('Tag: ') && this.count++;
```

Comment:
I don't think we'll want to prepend Tag: to every single tag name. I've been thinking about the best way to tell the difference between tags and foods and I think we can either consider it a tag if it has children and a food if it does not, or we can add an `isTag` field to every node.

Reply:
I think `isTag` would be preferable because then it would be easier to restrict food to only be in tags, unless it is okay if a food can have sub foods.